plugins/kubernetes/app/assets/javascripts/kubernetes/directives/kubernetes_deploy_progress_directive.js

```diff
@@ -0,0 +1,62 @@
+samson.directive('deployProgressWidget', function() {
+  return {
+    restrict: 'E',
+    templateUrl: 'kubernetes/deploy_progress_widget.tmpl.html',
+    scope: {
+      role: '=',
+      deployGroup: '='
+    },
+    controller: function($scope) {
+
+      $scope.currentRelease = function(release) {
+        var current = $scope.lastRelease();
```

Comment:
What happens if this is the first release ever for the project? Is this function called?